h2o-probes.d

```diff
@@ -62,12 +62,20 @@ provider h2o {
     /**
      * HTTP/1 server-level event, indicating that a connection has been accepted.
      */
-    probe h1_accept(uint64_t conn_id, struct st_h2o_socket_t *sock, struct st_h2o_conn_t *conn);
+    probe h1_accept(uint64_t conn_id, struct st_h2o_socket_t *sock, struct st_h2o_conn_t *conn, struct timeval *connected_at);
```

Comment:
FWIW I use a pointer for `struct timeval` just because BCC is not able to handle a 128-bit value object, which tires to use `uint128_t` internally, but there's no `uint128_t` in LLVM-BPF.